src/flint/types/fmpz.pyx

```diff
@@ -528,8 +528,8 @@ cdef class fmpz(flint_scalar):
         return u
 
     # This is the correct code when fmpz_or is fixed (in flint 3.0.0)
-    #
-    #def __or__(self, other):
+
+    # def __or__(self, other):
```

Comment:
Now that Flint 3.0 is the minimum version we could use this code instead.

Reply:
Ahh ok. Shall I make this change in this PR or another

Reply:
It should work fine just to uncomment this as it is tested. There was a bug in Flint 2.7 at the time.

Reply:
ok, this has been addressed